content/manuals/build/building/cdi.md

```diff
@@ -0,0 +1,531 @@
+---
+title: Container Device Interface (CDI)
+weight: 60
+description: Using CDI to access GPUs and other devices in your builds
+keywords: build, buildkit, buildx, guide, tutorial, cdi, device, gpu, nvidia, cuda, amd, rocm
```

Comment:
Looks like vale may need to be taught some of these acronyms (`nvidia`, `coda` ..) or would it be because they're lowercase here?

Reply:
I think keywords should be excluded from Vale analysis

Reply:
We need to revisit our Vale configuration to include a few more terms to the "allowlist". Yea, excluding metadata could be a good option.